core/src/test/java/smile/classification/SVMTest.java

```diff
@@ -30,6 +30,8 @@
 import smile.math.kernel.GaussianKernel;
 import smile.math.kernel.LinearKernel;
 import smile.math.kernel.PolynomialKernel;
+import smile.regression.ElasticNet;
```

Comment:
The equivalent SVM of ElasticNet is not this straightforward. Let's skip this for now.